ui/app/templates/components/job-page/parts/title.hbs

```diff
@@ -25,6 +25,15 @@
         @awaitingConfirmation={{this.stopJob.isRunning}}
         @onConfirm={{perform this.stopJob}} />
     {{else}}
+      <TwoStepButton
```

Comment:
Stop & Start didn't have any cancan stuff associated with them, so I felt decent about not adding any on this.

Maybe all of this should be wrapped in a "submit-job" check?